_test_/index.test.tsx

```diff
@@ -0,0 +1,7 @@
+﻿import React from 'react';
+import {render} from '@testing-library/react';
+import Home from "../pages";
+
+test('Check that the home page renders',()=>{
+    let home=render(<Home/>);
```

Comment:
Could you just add an expectation that page renders?
Currently the description of the test is a bit inaccurate... 
Looks good otherwise! :)